org.mixedrealitytoolkit.input/Visualizers/RiggedHandVisualizer/RiggedHandMeshVisualizer.cs

```diff
@@ -346,78 +228,15 @@ private float JointError(Vector3 armatureJointPosition, Vector3 userJointPositio
             return Vector3.Dot((armatureJointPosition - userJointPosition), fingerVector);
         }
 
-        private bool ShouldRenderHand()
+        protected override bool ShouldRenderHand()
         {
             // If we're missing anything, don't render the hand.
```

Comment:
Comment exists for code that no longer exists?

Reply:
I think it's still valid! We won't render if we're missing a hand subsystem, a wrist transform, a hand renderer, etc.

Reply:
I think I was referring to a different comment that appeared orphaned, but looking at the latest it looks fine.